zookeeper-server/pom.xml

```diff
@@ -190,6 +190,11 @@
       <artifactId>commons-io</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>dnsjava</groupId>
+      <artifactId>dnsjava</artifactId>
+      <version>3.5.1</version>
```

Comment:
The version should be defined in the root-level `pom.xml` file like for other third party libraries.